pkg/repo/fs.go

```diff
@@ -360,7 +359,7 @@ func (fsr *FsRepo) WriteRunInfo(ctx context.Context, summaryShellVariablesString
 	}
 
 	return runInfoPath, nil
-	// TODO previous behaviour put it in these places, we may consider creating a symlink later
```

Comment:
Isn't this the correct UK English spelling? We have a mix of behaviour vs behavior spelling in the codebase..

Reply:
Until recently I hadn't realized/realised so many words could be spelled/spelt differently. Not sure what the best practice/practise ought to be, perhaps just a good sense of humor/humour while we analyze/analyse this and maneuver/manoeuvre things? (sorry, I stop now...)